doc/01-usage.md

```diff
@@ -149,6 +149,96 @@ error: options validation failed for image type "iot-raw-xz": ostree.url: requir
 $
 ```
 
+### bootc
+
+`image-builder` supports building images from [bootable containers](https://docs.fedoraproject.org/en-US/bootc/getting-started/). Building bootc-based images works differently from `ostree`-based images and package-based images.
+
+When building a bootable container into an image we try to base everything on the container. Thus the distribution that is being built is not known; you cannot use the `--distro` argument in combination with `--bootc-*` arguments as it would do nothing.
+
+The container(s) used for the various `--bootc-*` arguments must be in container storage for the user you're running `image-builder` as before you start the build. This design choice was made due to the pulling of containers often needing additional authentication that `image-builder` cannot present the user with.
```

Comment:
Wording is a bit awkward here (first sentence, mostly).  Let's try moving it around a bit:

```suggestion
The container(s) used for the various `--bootc-*` arguments must be in the container storage of the user running `image-builder` before the start of the build. This avoids needing to configure `image-builder` with appropriate credentials or access rights for container registries.
```